lib/mirage/factories/cycle.ts

```diff
@@ -1,14 +1,19 @@
 import faker from 'faker'
 import { Factory } from 'miragejs'
 
-import { CADENCE } from 'src/components/Cycle/constants'
-
-export default Factory.extend({
-  period: faker.random.word,
-  active: faker.random.boolean,
-  cadence: () => faker.helpers.randomize(Object.values(CADENCE)),
+export const cycle = Factory.extend({
+  createdAt: faker.date.past(),
+  period: 'blablabla',
+  cadence: () => faker.helpers.randomize(['YEARLY', 'QUARTERLY']),
+  active: true,
   dateStart: faker.date.past,
   dateEnd: faker.date.future,
-  createdAt: faker.date.past,
-  updatedAt: faker.date.past,
+  updatedAt: faker.date.recent,
+  afterCreate(cycle, server) {
+    const status = server.create('status')
+    const delta = server.create('deltum') // Miragejs tries to singularize the word delta, incorrectly
```

Comment:
🌫 **Dust** - You could add the link to the [issue I've opened](https://github.com/martinandert/inflected/issues/25) too